site/en/adminGuide/upgrade.md

```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
+milvus/milvus        2.2.5                2.0.0-rc.7                Milvus is an open-source vector database built ...
+milvus/milvus        2.2.4                2.0.0-rc.7                Milvus is an open-source vector database built ...
+milvus/milvus        2.2.3                2.0.0-rc.7                Milvus is an open-source vector database built ...
+milvus/milvus        2.2.2                2.0.0-rc.7                Milvus is an open-source vector database built ...
+milvus/milvus        2.2.1                2.0.0-rc.6                Milvus is an open-source vector database built ...
+milvus/milvus        2.2.0                2.0.0-rc.6                Milvus is an open-source vector database built ...
 ```
 
-You can see there are several new versions after **v2.0.0-rc4**.
+### 5. Upgrade
 
-5. Upgrade to **v2.0.0-rc5-hotfix1**:
+Run the following commands to upgrade your Milvus version from v2.0.0-rc7 to v2.0.0-rc8.
 
 ```
-helm repo update
-helm upgrade my-release milvus/milvus
-helm list
-NAME              NAMESPACE        REVISION        UPDATED                                     STATUS          CHART                APP VERSION
-my-release        default          2               2021-09-06 15:01:24.570561 +0800 CST        deployed        milvus-2.1.14        2.0.0-rc.5-hotfix1
+$ helm repo update
+$ helm upgrade my-release milvus/milvus --set cluster.enabled=false --set etcd.replicaCount=1 --set minio.mode=standalone --set pulsar.enabled=false 
 ```
 
-You can see the new pods:
+Run `$ helm list` again to check your Milvus version. You can see your Milvus standalone has been updated to v2.0.0-rc8.
+
+```
+NAME              NAMESPACE        REVISION        UPDATED                                     STATUS          CHART               APP VERSION
+my-release        default          2               2021-11-08 17:15:46.530627 +0800 CST        deployed        milvus-2.3.3        2.0.0-rc.8
+```
+
+Run `$ kubectl get pods` to check the new pods. You can see the following output.
 
 ```
-kubectl get pods
 NAME                                            READY   STATUS    RESTARTS   AGE
-my-release-etcd-0                               1/1     Running   0          46m
-my-release-milvus-standalone-546649bcdf-xqjd5   1/1     Running   0          31m
-my-release-minio-744dd9586f-drjnr               1/1     Running   0          31m
+my-release-etcd-0                               1/1     Running   0          3m32s
+my-release-milvus-standalone-6967454987-72r55   1/1     Running   0          22s
+my-release-minio-744dd9586f-qngzv               1/1     Running   0          3m32s
 ```
 
 <div class="alert note">
-To upgrade Milvus standalone, old pods will be deleted first. Therefore, the service may be offline for a short period of time.
+When upgrading Milvus standalone version, old pods will be deleted. Therefore, the service may be offline for a short period of time.
```

Comment:
I don't think you need "version" with "upgrading."
"When upgrading Milvus standalone" is good enough. 
Up to you anyway.